shared/js/ui/templates/shared/tracker-networks-text.es6.js

```diff
@@ -1,22 +1,27 @@
 const bel = require('bel')
 
-module.exports = function (site, isMajorNetworksCount) {
-  const trackerNetworksCount = isMajorNetworksCount ? site.majorTrackersCount : site.totalTrackersCount
+module.exports = function (site, isMajorNetworksCount, includeUniqueTrackersCount) {
+  const trackerNetworksCount = isMajorNetworksCount ? site.majorTrackerNetworksCount : site.totalTrackerNetworksCount
 
   let trackersText = isMajorNetworksCount ? ' Major Tracker' : ' Tracker'
   trackersText += (trackerNetworksCount === 1) ? ' Network ' : ' Networks '
+  let finalText = trackerNetworksCount + trackersText + trackersBlockedOrFound(site, trackerNetworksCount)
 
-  const finalText = trackerNetworksCount + trackersText + trackersBlockedOrFound(site, trackerNetworksCount)
+  if (includeUniqueTrackersCount && trackerNetworksCount > 0) {
+    const uniqueTrackersText = site.trackersCount === 1 ? ' Unique Tracker In ' : ' Unique Trackers In '
+    finalText = site.trackersCount + uniqueTrackersText + finalText
+    return bel`${finalText}`
```

Comment:
Tiny nitpick, this `return` is unnecessary.

Reply:
True True